### PR TITLE
gscan: fix jumping status icons

### PR DESCRIPTION
**Pull request by oliver-sanders**
Fixes an issue in gscan where the status icons can jump back and forth when hovered over providing that the cycle points of the suite have been expanded.

Uses a blank pixbuf rather than setting `visible` to `False` for hiding error icons where they are not needed / wanted.

@arjclark Please Review / Assign second if desired

### CODE REVIEW

**Comment by matthewrmshin** [comment]
I think we should have a back port of this to 6.11.x as well.

**Comment by arjclark** [comment]
Review1: Looks good to me, manually tested as no longer having the problem.

@matthewrmshin - please review 2.

File: lib/cylc/gui/gscan.py [hunk 1]
```diff
@@ -485,8 +485,8 @@ def __init__(self, hosts=None, owner=None, poll_interval=None):
             self.warn_icon_grey, 0, False)  # b&w warn icon pixbuf
         status_column.pack_start(warn_icon, expand=False)
         status_column.set_cell_data_func(warn_icon, self._set_error_icon_state)
-        warn_icon.set_property('visible', False)
-
+        self.warn_icon_blank = gtk.gdk.Pixbuf(  # Transparent pixbuff.
+            gtk.gdk.COLORSPACE_RGB, True, 8, 17, 17).fill(0x00000000)
```
Comment (matthewrmshin) [inline lib/cylc/gui/gscan.py]: 8, 17, 17? (I can see 17, 17 a few lines up.) Probably worth using a constant or two?

Comment (arjclark) [inline lib/cylc/gui/gscan.py]: Potentially they could be extracted from warn_icon?

Comment (oliver-sanders) [inline lib/cylc/gui/gscan.py]: Tis the width/height of the pixbuf icons normally implied from the xpm strings. I'll stick it in a constant here for this module.

**Pull request merged**